Workarounds for Yarn not playing nicely on Windows

soundasleep opened the pull request:
- Bypass `yarn run` when running on Windows; instead, we can go directly to `node_modules/.bin/`
- Create batch scripts so users don't need to type in `ruby bin/webpack` manually

See discussion on https://github.com/rails/webpacker/pull/583

gauravtiwari commented:
@soundasleep Looks great 👍  🍰  Not sure about `.bat` scripts though. How do you run `rails` binstub on windows? `./bin/rails` or `ruby ./bin/rails`?

Can we also document this in README and Changelog please?

gauravtiwari commented:
Lets drop `.bat` scripts (primarily because it's not uniform with other binstubs and isn't useful for non-window users). We can document this in README to use it as `ruby ./bin/webpack`

soundasleep commented:
Yup! That's a great idea. Updated the MR, and removed the batch scripts.

On Windows you don't tend to use any of the binstubs anyway, you'll use `rails` directly, as there's usually only a single version globally installed. e.g. http://railsinstaller.org/en

gauravtiwari commented:
Awesome 🍰  🎉  Looks great, thank you :)

Yeah, that makes sense 👍
